src/vue/components/morph-parts/morph-data.vue

```diff
@@ -6,7 +6,7 @@
         <div class="alpheios-morph-data__morphdata" v-if="hasMorphData">
             <span class="alpheios-morph-data__pofs">
                 <inflectionattribute v-for="(feat, featIndex) in featuresList.pofs" v-bind:key="featIndex"
-                    :data="lexeme.lemma.features" :type="types[feat]"
+                    :data="lexeme.lemma.features" :type="feat"

```

Comment:
I'm not sure about this change. In theory, it should still be doing the exact same thing. I.e. `types[feat]` would have returned 'case' where `feat === 'case'` and 'part of speech` where `feat === 'part'

One could argue that this is unnecessarily complicated, and I am not sure I can unwind why we did it this way in the first place, but looking at the code, I'm not sure how this change has any affect.

Reply:
Oh you are right!
For now (without this change) - we have two inconsistent places, that was the source of the  problem :
![image](https://user-images.githubusercontent.com/12377640/72944851-eea71380-3dc5-11ea-857c-7ec9b740baf9.png)

here in components attributes - types[feature name]
and in code part - here we use feature[feature name]

I have unified to the second variant, because 
  - it has more usage in this component
  - we use direct names (without Feature.types) for infl-attributes in other places

so it  is our usual practice. 

But I agree that usage of Feature.types is a little more complicated but could be useful in some situations.
So I would change to this base.

Reply:
Fix are made